pkg/crd/schema.go

```diff
@@ -291,6 +291,12 @@ func mapToSchema(ctx *schemaContext, mapType *ast.MapType) *v1beta1.JSONSchemaPr
 		valSchema = localNamedToSchema(ctx.ForInfo(&markers.TypeInfo{}), val)
 	case *ast.SelectorExpr:
 		valSchema = namedToSchema(ctx.ForInfo(&markers.TypeInfo{}), val)
+	case *ast.ArrayType:
+		valSchema = arrayToSchema(ctx.ForInfo(&markers.TypeInfo{}), val)
+		if valSchema.Type == "array" && valSchema.Items.Schema.Type != "string" {
```

Comment:
Is it possible `valSchema.Items.Schema` may be nil?
If possible, we should check it. Otherwise, you can ignore this comment.

Reply:
It is not possible if `valSchema.Type != "array"`